src/App/Pages/Vault/VaultListGroupingsPage.cs

```diff
@@ -148,7 +149,14 @@ private void Init()
             AddCipherItem?.InitEvents();
             SearchItem?.InitEvents();
 
-            _filterResultsCancellationTokenSource = FetchAndLoadVault();
+            if(lastTappedItem != null)
+            {
+                ListView.ScrollTo(lastTappedItem, ScrollToPosition.Center, false);
+            }
+            else
+            {
+                _filterResultsCancellationTokenSource = FetchAndLoadVault();
```

Comment:
Won't this now cause the list to never update when changes are made on the add/edit page and you go back?

Reply:
Ah, I didn't think of editing it...

I couldn't figure out how to get the ScrollTo() call to work in the FetchAndLoadVault() function. Even if I call it when the content is set to the ListView the refresh still happens afterwards and jumps to the top.

I'll keep digging, somehow the ScrollTo() needs to be called after FetchAndLoadVault() returns.

Reply:
This must only be an issue on UWP because the position is remembered correctly on ios and android. Maybe it's because we use modals on those and you arn't on UWP?

Reply:
Hmmm... But we need not modals to get the back button